src/Adapter/FilesystemItems.php

```diff
@@ -135,7 +135,7 @@ public function list(int $limit = 10, ?string $cursor = null) : ItemListPage
             $found = null;
             $i = 0;
             foreach ($ids as $id) {
-                if ($id == $cursor) {
+                if ($id->equals($cursor)) {
```

Comment:
This is the 'downside' to https://github.com/libero/php-coding-standard/pull/20.

Reply:
Welcome to Java?

Reply:
What adds to the confusion here however is that `$cursor` changes type from `string` to `ItemId`

Reply:
Agreed, that's not good. Will rename.